addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3592,11 +3593,6 @@ export class OdooEditor extends EventTarget {
         this._fixFontAwesomeSelection();
 
         this._fixSelectionOnContenteditableFalse();
-
-        this.historyUnpauseSteps('handleSelectionInTable');
```

Comment:
This has disappeared, is this intended ? I would guess no since I suspect the corresponding pauseSteps is still present somewhere ? You might want to group the two "mousup" fix PR together as they will conflict anyway.